proposals/improved-interpolated-strings.md

```diff
@@ -138,15 +153,21 @@ to be converted to an _applicable\_interpolated\_string\_builder\_type_. There a
 1. A method argument is converted as part of determining applicable function members (covered below), or
 2. Given an _interpolated\_string\_expression_ `S` being converted to type `T`, the following is true:
     * `T` is an _applicable\_interpolated\_string\_builder\_type_, and
-    * `T` has an accessible static bool-returning method `GetInterpolatedStringBuilder` that takes 2 int parameters and 1 out parameter of type `T`, in that order.
+    * `T` has an accessible static bool-returning method `Create` that takes 2 int parameters and 1 out parameter of type `T`, in that order.
 
-We want to make `GetInterpolatedStringBuilder` a static method with an `out` parameter for 2 reasons:
+We want to make `Create` a static method with an `out` parameter for 2 reasons:
 
 1. By making it a `static` method instead of a constructor, we allow the implementation to pool builders if it so decides to. If we limited the pattern to constructors,
 then the implementation would be required to always return new instances.
-2. By making the builder an `out` parameter we allow the `GetInterpolatedStringBuilder` method to return a bool indicating whether to continue formatting, which is useful
+2. By making the builder an `out` parameter we allow the `Create` method to return a bool indicating whether to continue formatting, which is useful
 for scenarios like the logger above that may want to skip any argument evaluation at all for cases when the log level isn't enabled.
 
+**Open Question**
+
+Should we swap the out parameter and return? IE, make the return the builder type, and have an optional bool out parameter for whether the builder was successfully created? This
+might be easier for authoring and have better perf characteristics. While it does differ from the standard `TryX` pattern in C#, this method isn't intended for direct usage, so
+diverging may be fine.
```

Comment:
The current form also means you need to spend more time specing out `ref struct` lifetime issues. I'm happy to sit down and work with you on this but I'd prefer to pick the form that doesn't require us to do this :smile: